docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,173 @@ This section provides instructions for migrating your hosts away from your old M
+   
+4. Once this setting has been approved, the MDM enrollment profile cannot be removed by the end user.
+
+##### How to turn on MDM (default)
+
+1. Select the Fleet icon in your menu bar and select **My device**.
+
+![Fleet icon in menu bar](https://raw.githubusercontent.com/fleetdm/fleet/main/website/assets/images/articles/fleet-desktop-says-hello-world-cover-1600x900@2x.jpg)
+
+2. On your **My device** page, select **Turn on MDM** the button in the yellow banner and follow the instructions. 
+    * If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. 
+    * If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+<img width="1400" alt="My device page - turn on MDM" src="https://user-images.githubusercontent.com/5359586/229950406-98343bf7-9653-4117-a8f5-c03359ba0d86.png">
+
+#### End user workflow
 
 > Available in Fleet Premium or Ultimate
 
-You can use Fleet's end user migration workflow to reduce the gap in MDM coverage during migration.
+The end user migration workflow is supported for automatically enrolled (DEP) hosts.
+
+To watch a GIF that walks through the end user experience for the end user migration workflow, in the Fleet UI, head to **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+In Fleet, you can configure the end user workflow using the Fleet UI or fleetctl command-line tool.
+
+Fleet UI:
+
+1. Select the avatar on the right side of the top navigation and select **Settings > Integrations > Mobile device management (MDM)**.
 
-The migration worfklow is supported for automatically enrolled (DEP) hosts.
+2. Scroll down to the **End user migration workflow** section and select the toggle to enable the workflow.
 
-During the end user migration workflow, an end user's device will have their selected system
-theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds,
-you can optionally set an alternate logo for the themes.
+3. Under **Mode** choose a mode and enter the webhook URL for you automation tool (ex. Tines) under **Webhook URL** and select **Save**.
 
-You can do this in the Fleet UI by going to **Settings** > **Organization settings** >
-**Organization info** and adding a url to the desired image in the **Organization avatar URL (for
-dark backgrounds)** and **Organization avatar URL (for light backgrounds)** inputs. The appropriate
-image will show depending on the selected system theme.
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
```

Comment:
"both light and dark" might be clearer. Also, may consider an alternative to "doesn't look good" just to avoid negative-sounding language about the user's logo:
```suggestion
4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo is not easy to see on both light and dark backgrounds, you can optionally set a logo for each theme:
```